clients/fides-js/src/lib/i18n/i18n-utils.ts

```diff
@@ -72,11 +131,28 @@ function extractMessagesFromExperienceConfig(
       // For each translation, extract each of the translated fields
       (translation: ExperienceConfigTranslation) => {
         const locale = translation.language;
+        // We only override experience translations if the override_language matches current locale
+        let localeHasOverride = false;
+        if (experienceTranslationOverrides?.override_language) {
+          localeHasOverride = !!matchLocale(
+            experienceTranslationOverrides?.override_language,
+            [locale]
+          );
+        }
         const messages: Messages = {};
         EXPERIENCE_TRANSLATION_FIELDS.forEach((key) => {
+          let overrideValue: string | null | undefined = null;
+          if (experienceTranslationOverrides) {
```

Comment:
```suggestion
          if (experienceTranslationOverrides && localeHasOverride) {
```

It's just a bit more legible like this